python/src/converters/__init__.py

```diff
@@ -3,3 +3,4 @@
 
 def register_converters():
     special_converters.append(MatchIteratorPairConverter())
+    special_converters.append(OutputStdStringUnicodeConverter())
```

Comment:
we should try to get this into autowrap.

Having that said, as we do not know if and when they accept it, this is of course fine for now.

Reply:
Sure, will send a PR to them as well.